xbmc/cores/VideoPlayer/VideoRenderers/VideoShaders/WinVideoFilter.h

```diff
@@ -144,7 +156,8 @@ class CYUV2RGBShader : public CWinShader
   CPoint              m_dest[4];
   EBufferFormat       m_format;
   float               m_texSteps[2];
-  COutputShader *m_pOutShader;
+  COutputShader      *m_pOutShader;
+  bool                m_isNearest;
```

Comment:
No need to touch the existing line, just add your variable without the additional whitespace. We discourage this kind of alignment, even though I use it in some of the code I maintain.